ci-operator/config/openshift/api/openshift-api-release-4.12.yaml

```
@@ -49,7 +49,7 @@ tests:
 - as: verify-client-go
   commands: |
     dir=$GOPATH/src/github.com/openshift/client-go
-    git clone https://github.com/openshift/client-go $dir
+    git clone https://github.com/openshift/client-go --branch release-4.12 --single-branch $dir
```

Prow injects `PULL_BASE_REF` into every pod it creates for tests, this would point to the correct branch, how about the following and updating all releases to use the same?
```suggestion
    git clone https://github.com/openshift/client-go --branch ${PULL_BASE_REF:-master} --single-branch $dir
```

---

OK good catch. Thanks. Do you want me to update all releases in this PR?

---

I've just triggered a rehearsal, if that passes, could you update every branch from 4.12 onwards so that we keep the change for newer release

---

OK

---

Ahh that's a little awkward, the `PULL_BASE_REF` for the rehearsal is `master`, so it presented the same result as before. To prove this is working we would have to merge and see what happens when we run against a backport

---

hmm it seems it doesn't work right